.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "[js]": {
+    "editor.defaultFormatter": "standard.vscode-standard",
+    "editor.formatOnSave": true
```

Comment:
I wonder if `formatOnSave` should still be a user level setting? I have no strong opinions on it though, just took me a long time to switch to 'on save' actions!

Reply:
Probably actually, given how many different developers we have! Without it specified here, it will obviously still respect your own `formatOnSave` being on